utils/ccloud_library.sh

```diff
@@ -517,7 +517,7 @@ function ccloud::wait_for_connector_up() {
 
   connectorName=$(cat $filename | jq -r .name)
   echo "Waiting up to $maxWait seconds for connector $filename ($connectorName) to be RUNNING"
-  retry $maxWait ccloud::validate_connector_up $connectorName || exit 1
+  ccloud::retry $maxWait ccloud::validate_connector_up $connectorName || exit 1
```

Comment:
In order differentiate from the function in the other library, WDYT about renaming this

from `ccloud::retry` to `ccloud::retry_until`?

Reply:
It is distinquished by the pseudo namespace.  If we rename the function in one place but not in the other does it seem to a reader it does something different?

Reply:
Good point.  It's just a weird situation having the same functionality in two different places, but current implementation is probably the best path forward, thanks!